source/Octopus.Server.Client/Repositories/Async/ProjectRepository.cs

```diff
@@ -268,9 +268,10 @@ public Task<ProgressionResource> GetProgression(ProjectResource project)
             return GetProgression(project, CancellationToken.None);
         }
 
-        public Task<ProgressionResource> GetProgression(ProjectResource project, CancellationToken cancellationToken)
+        public Task<ProgressionResource> GetProgression(ProjectResource project, CancellationToken cancellationToken, int? releaseHistoryCount = null)
```

Comment:
Even though the overload without a cancellation token is marked `[Obsolete]`, we should keep parity with the other overload by adding the `releaseHistoryCount` parameter there too, as it's specifically the lack of a cancellation token that makes it obsolete.